jobbergate-api/jobbergate_api/apps/job_submissions/schemas.py

```diff
@@ -60,9 +60,223 @@
         description="The report message received from cluster-agent when a job submission is rejected",
         example="Unrecognized SBATCH arguments",
     ),
+    execution_parameters=MetaField(
+        description=(
+            "The parameters to be passed to the job submission. "
+            "See more details at: https://slurm.schedmd.com/rest_api.html#v0.0.36_job_properties"
```

Comment:
This won't work because schdmd only shows on the website the most up to date version (right now it is 0.0.38). I think just pointing to  https://slurm.schedmd.com/rest_api.html is fine

Reply:
fixed in [8d49b6b](https://github.com/omnivector-solutions/jobbergate/pull/194/commits/8d49b6b0f4c045df2942165f86e8b578ea9a0463)